models/chainlink/arbitrum/chainlink_arbitrum_ccip_fulfilled_transactions.sql

```diff
@@ -1,39 +1,94 @@
 {{
   config(
-    tags = ['prod_exclude'],
+    
```

Comment:
as part of a universal cleanup effort, please move `schema` property from `dbt_project` file to models directly in each config block